api/src/main/java/org/apache/iceberg/Table.java

```diff
@@ -263,6 +263,17 @@ default AppendFiles newFastAppend() {
    */
   DeleteFiles newDelete();
 
+  /**
+   * Create a new {@link UpdateTableStatistics update table statistics API} to add or remove
+   * statistics files in this table.
+   *
+   * @return a new {@link UpdateTableStatistics}
+   */
+  default UpdateTableStatistics updateTableStatistics() {
```

Comment:
I think it is sufficient to call this `updateStatistics`. Is there a reason to qualify it with `table`?

Reply:
ok